src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/commands.py

```diff
@@ -107,8 +107,7 @@ def get_custom_sdk(custom_module, client_factory, resource_type=ResourceType.DAT
         g.storage_command_oauth('snapshot', 'snapshot_blob')
         g.storage_command_oauth('update', 'set_blob_properties')
         g.storage_command_oauth('exists', 'exists', transform=create_boolean_result_output_transformer('exists'))
-        g.storage_command_oauth('delete', 'delete_blob', transform=create_boolean_result_output_transformer('deleted'),
-                                table_transformer=transform_boolean_for_table)
+        g.storage_command_oauth('delete', 'delete_blob')
```

Comment:
We need to add the appropriate history entry for this. This would also be a breaking change because we are changing the return structure of the command.